avalanche/training/plugins/rwalk.py

```diff
@@ -0,0 +1,206 @@
+import warnings
+
+import torch
+import torch.nn.functional as F
+
+from avalanche.training.plugins.strategy_plugin import SupervisedPlugin
+from avalanche.training.utils import copy_params_dict, zerolike_params_dict
+from avalanche.models.utils import avalanche_forward
+
+
+class RWalkPlugin(SupervisedPlugin):
```

Comment:
Maybe RWalkImportancePlugin or EWC++Plugin? To clarify what it is doing.

Reply:
In the paper they refer to RWalk as the penalty/regularization term (see the title of the third paragraph of sect. 4.1). The sampling strategies are later introduced in sect. 4.2. In any case, I'll add a warning in the docs!

Reply:
thanks, that should be enough